src/test/java/org/sagebionetworks/bridge/models/accounts/UserSessionTest.java

```diff
@@ -53,7 +68,8 @@ public void canSerialize() throws Exception {
             .withHealthCode("healthCode")
             .withSharingScope(ALL_QUALIFIED_RESEARCHERS)
             .withRoles(ImmutableSet.of(ADMIN))
-            .withDataGroups(ImmutableSet.of("group1", "group2")).build();
+            .withDataGroups(ImmutableSet.of("group1", "group2"))
+            .withEnrollments(enrollments).build();
```

Comment:
Again, more tests of the JSON

Reply:
In this case, we test that the study participant is equals in the newly deserialized session, and this includes the enrollments.